### DISCUSSION TITLE
Game ID X Game Name

### DISCUSSION THREAD

**Discussion by JoeHouser** [mtkennerly/ludusavi#414] [Q&A]
for some reason my ludusavi doesn't show the name of my games, but it shows the id of the games
![image](https://github.com/user-attachments/assets/70dfaa2e-7099-43d4-a7dd-edfa4ae7d0a4)

**Comment by mtkennerly** [comment]
Hi! Do you use Hydra? It looks like you've been affected by this issue: https://github.com/hydralauncher/hydra/issues/1202

To fix this, you would need to go to the "other" screen in Ludusavi, re-enable the primary manifest, and disable the secondary manifest added by Hydra. Note that you'll have to fix this every time you open Hydra, because it changes Ludusavi's settings automatically.
